### ISSUE TITLE
Execute any emote locally

### ISSUE THREAD

**Issue by karashiiro** [goatcorp/suggestions#228]
### Description
See title.

### More info
See https://discord.com/channels/581875019861328007/685275026156683280/795304958337351690 for more information.

Original requester: Phi#0999

----

 - [X] I checked if this request wasn't already proposed
 - [X] I made sure this request follows Dalamud's [plugin guidelines](https://github.com/goatcorp/FFXIVQuickLauncher#isnt-this-cheating)

**Comment by johnysandels** [comment]
Possible in CMTool, and also Franz said "many devs here aren't interested in cheating tools or bypassing purchased content"

**Comment by karashiiro** [comment]
Would like a yes/no from @goaaats on this one, it'd only be local if implemented according to spec, so it should be unobjectionable.

**Comment by goaaats** [comment]
Technically, rules-wise, it's not objectionable, but circumventing "paywalls" is not something I'd encourage for the reasons Franz mentioned. AFAIK, TexTools bans this as well from their official repositories.

Generally, for most things CMTool implements, I think relying on CMTool/Anamnesis instead is best.

**Comment by karashiiro** [comment]
any updates?

**Comment by TheGokki** [comment]
I would agree that it would be really nice to be able to play any emotes. That said - bypassing regular ways when it comes to paid content or cheating by playing content locked behind progression, such as Eureka or Deep Dungeon emotes would not be ideal.

However, i think a happy medium would be nice to have: do not play emotes normally, instead - have it integrate through Mare Synchronous and have them play over Mare instead of locally. This way, a normal player wouldn't see those emotes, but would still work through Mare.

**Comment by tommadness** [comment]
That's the definition of bypassing content/paywalls, not a "happy medium".

**Comment by reiichi001** [comment]
@TheGokki. I don't think you've quite understood the request. If you can't even play an emote locally, why would it then broadcast over a plugin we provide no official support for?

This request was never about letting other people see an emote you rightfully cannot ask the server to play for other players.

The only way this plugin request works is if it's local only because you have to bypass the unlock flags, which means nobody else can see that you've used an emote you don't technically own/have unlocked.

**Comment by TheGokki** [comment]
Then there's no point to this i guess, sorry about that.
